recipes/mpi4py-fft/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set build = 0 %}
 {% set name = "mpi4py-fft" %}
-{% set version = "1.0.3" %}
```

Comment:
The last commit updated the version but not the sha256. Does the checksum also need updating?

Reply:
Excellent point.